### PR TITLE
reduced max-active connections for connection pool

### PR DESCRIPTION
**Pull request by patduin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## TBD
+### Changed
+* Changed default max-active connection pool size from 50 to 2 it was unnecessarily large and having multiple housekeeping jobs was causing RDS issues (too many open connections)
```
Comment (jmnunezizu) [inline CHANGELOG.md]: There's a full stop missing :)

File: housekeeping/src/main/java/com/hotels/housekeeping/HousekeepingConfiguration.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ public void postConstruct() {
         .put("spring.jpa.properties.hibernate.default_schema", "${housekeeping.schema-name:housekeeping}")
         .put("spring.datasource.initialize", true)
         .put("spring.datasource.max-wait", 10000)
-        .put("spring.datasource.max-active", 50)
+        .put("spring.datasource.max-active", 2)
```
[the remark this answers is not in the document]
Comment (patduin) [reply housekeeping/src/main/java/com/hotels/housekeeping/HousekeepingConfiguration.java]: hang on you are right, we might be overwriting it here.

File: CHANGELOG.md (hunk 1 above)
Comment (patduin) [reply CHANGELOG.md]: doh

File: housekeeping/src/main/java/com/hotels/housekeeping/HousekeepingConfiguration.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (patduin) [reply housekeeping/src/main/java/com/hotels/housekeeping/HousekeepingConfiguration.java]: ok checked an no it isn't overwritten here, this is added with lowest prio, so overriding in yml is perfectly ok.

[the remark this answers is not in the document]
Comment (patduin) [reply housekeeping/src/main/java/com/hotels/housekeeping/HousekeepingConfiguration.java]: @jmnunezizu hikariCp says this on `minimumIdle`
> This property controls the minimum number of idle connections that HikariCP tries to maintain in the pool. If the idle connections dip below this value and total connections in the pool are less than maximumPoolSize, HikariCP will make a best effort to add additional connections quickly and efficiently. However, for maximum performance and responsiveness to spike demands, we recommend not setting this value and instead allowing HikariCP to act as a fixed size connection pool. Default: same as maximumPoolSize

We don't override it, but dropping max to 2 should fix this.

[the remark this answers is not in the document]
Comment (jmnunezizu) [reply housekeeping/src/main/java/com/hotels/housekeeping/HousekeepingConfiguration.java]: > Default: same as maximumPoolSize

No wonder why with 50 connections it was hogging our DB cluster!

**Comment by patduin** [comment]
@jmnunezizu if you're ok with this PR could you approve please?

**Pull request merged**